spec/models/pet_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe Pet, type: :model do
   describe "relationships" do
     it { should belong_to(:shelter) }
+    it { should have_many :pet_applications}
```

Comment:
Make sure you add this relationship test as well
it { should have_many(:applications).through(:pet_applications) }